### PR TITLE
DSD-1281: Fix Hero backgroundColor/foregroundColor issue

### PR DESCRIPTION
**Pull request by jackiequach**
Fixes JIRA ticket [DSD-1281](https://jira.nypl.org/browse/DSD-1281)

## This PR does the following:

- Fixes the issue with `backgroundColor` and `foregroundColor` props not prioritizing the passed design token values for the `Hero` component. Chakra prioritizes `bg` over `backgroundColor` which caused the default value to overwrite the passed value.
- `foregroundColor` for the campaign variant was being overwritten by the default styles passed through `__css`, this was resolved by adding `contentBoxStyling` to the object.

## How has this been tested?

Locally in Storybook

## Accessibility concerns or updates

-

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Hero/Hero.tsx [hunk 1]
```diff
@@ -160,13 +160,13 @@ export const Hero = chakra(
           ? { bgColor: backdropBackgroundColor }
           : { backgroundColor };
       } else if (heroType === "tertiary" || heroType === "fiftyFifty") {
-        backgroundImageStyle = { backgroundColor };
+        backgroundImageStyle = backgroundColor ? { bg: backgroundColor } : {};
```
Comment (EdwinGuzman) [inline src/components/Hero/Hero.tsx:163]: `backgroundColor` is a css prop, but was the issue that `bg` has more precedence? Or perhaps a mix of this and that the `contentBoxStyling` came before the `styles.content` object?

Comment (jackiequach) [reply src/components/Hero/Hero.tsx:163]: It seems to be somewhat unknown, `bg` for content is overwritten by the `backgroundColor` added with `contentBoxStyling`, but `backgroundColor` does not overwrite `bg` in the parent component. By setting `backgroundImageStyle` to use `bg` instead, the spread operator overwrites the default value consistently.

**Review by EdwinGuzman — APPROVED** [review]

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  Just make sure to update the `Latest` version number on the `Hero` component story page.  Also, I will need some help writing the AC for this ticket.  Is there any way to validate this update through Storybook?

**Comment by EdwinGuzman** [comment]
True, @jackiequach can you add an example in the storybook doc or update the existing examples with the better design token values instead of hex values (if any are used).

**Comment by jackiequach** [comment]
Updated the last Campaign example with design token values.

**Review by EdwinGuzman — APPROVED** [review]
Looks good.

**Review by bigfishdesign13 — APPROVED** [review]
Looks good!

**Pull request merged**